packages/codemirror-blocks/src/ui/BlockEditor.tsx

```diff
@@ -135,36 +132,34 @@ const BlockEditor = ({ options = {}, ...props }: BlockEditorProps) => {
    * When the editor mounts, (1) set change event handlers and AST,
    * (2) set the focus, (3) set aria attributes, and (4) build the API
    */
-  const handleEditorDidMount = (editor: CodeMirrorFacade) => {
+  const handleEditorDidMount = (codemirror: CodeMirror.Editor) => {
+    const editor = new CodeMirrorFacade(codemirror);
     setEditor(editor);
     // TODO(Emmanuel): Try to set them in the component constructor
```

Comment:
This is the only place where we manually use CM's methods to attach handlers. It's ugly. I wasn't able to convince TypeScript that setting these in the constructor was a valid move. I suspect you'll have a better shot at figuring it out. :)